cmd/topicctl/subcmd/get.go

```diff
@@ -211,10 +221,10 @@ func membersCmd() *cobra.Command {
 }
 
 func partitionsCmd() *cobra.Command {
-	return &cobra.Command{
-		Use:   "partitions [topic]",
-		Short: "Displays partition information for the specified topic.",
-		Args:  cobra.ExactArgs(1),
+	partitionsCommand := &cobra.Command{
+		Use:   "partitions [topics]",
+		Short: "Get all partitions information for topics",
+		Args:  cobra.MinimumNArgs(0),
```

Comment:
This command needs an argument, right? Shouldn't this be `cobra.MinimumNArgs(1)`?